sklearn/covariance/tests/test_covariance.py

```diff
@@ -121,10 +121,10 @@ def test_ledoit_wolf():
                                               block_size=6),
                         shrinkage_)
     # compare shrunk covariance obtained from data and from MLE estimate
-    lw_cov_from_mle, lw_shinkrage_from_mle = ledoit_wolf(X_centered,
-                                                         assume_centered=True)
+    lw_cov_from_mle, lw_shrinkrage_from_mle = ledoit_wolf(X_centered,
```

Comment:
almost...

lw_shrinkrage_from_mle -> lw_shrinkage_from_mle

correct word is shrinkage